app/routes.php

```diff
@@ -50,6 +51,8 @@
             ->add(PlainRecaptchaMiddleware::class)
         ;
 
+        $versionGroup->get('/password-reset-token/{base58:[A-Za-z0-9-]{22}}', GetPasswordResetToken::class);
```

Comment:
I'd call it token for consistency:
```suggestion
        $versionGroup->get('/password-reset-token/{token:[A-Za-z0-9-]{22}}', GetPasswordResetToken::class);
```

Or 
```suggestion
        $versionGroup->get('/password-reset-token/{base58Token:[A-Za-z0-9-]{22}}', GetPasswordResetToken::class);
```